sirepo/package_data/static/js/controls.js

```diff
@@ -582,16 +582,74 @@ SIREPO.viewLogic('quadrupoleView', function(appState, controlsService, panelStat
     };
 });
 
+SIREPO.app.directive('optimizationPicker', function(latticeService) {
+    return {
+        restrict: 'A',
+        scope: {
+            controller: '='
+        },
+        template: [
```

Comment:
Can remove array and just have template string. See [this example](https://github.com/radiasoft/sirepo/blob/581f23a9f784dc51b5531cd69bfa66eceb73799a/sirepo/package_data/static/js/raydata.js#L224_).